core/src/main/java/org/pentaho/di/core/database/Database.java

```diff
@@ -2873,17 +2869,12 @@ public void setProcLookup( String proc, String[] arg, String[] argdir, int[] arg
   }
 
   public Object[] getLookup( PreparedStatement ps, boolean failOnMultipleResults ) throws KettleDatabaseException {
-    return getLookup( ps, failOnMultipleResults, false );
-  }
-
-  public Object[] getLookup( PreparedStatement ps, boolean failOnMultipleResults, boolean lazyConversion )
-    throws KettleDatabaseException {
     ResultSet res = null;
     try {
       log.snap( Metrics.METRIC_DATABASE_GET_LOOKUP_START, databaseMeta.getName() );
       res = ps.executeQuery();
 
-      Object[] ret = getRow( res, lazyConversion );
+      Object[] ret = getRow( res );
```

Comment:
Why was the specification of lazy conversion removed here ? I think it needs to be put back.